server/controllers/landingController.test.ts

```diff
@@ -88,9 +89,7 @@ describe('LandingController', () => {
 
       const requestHandler = landingController.index()
 
-      requestHandler(request, response, next)
-      expect(response.status).toHaveBeenCalledWith(403)
-      expect(response.render).toHaveBeenCalledWith('temporary-accommodation/static/userDetailsRequired')
+      expect(() => requestHandler(request, response, next)).toThrowError(UnauthorizedError)
```

Comment:
Nice -- good to see this can now leverage the existing logic for unauthorised access!